### PR TITLE
feat: dashboard saving

### PR DESCRIPTION
**Pull request by tracy-french**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

## Legal

This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

File: apps/client/src/services/generated/models/DashboardWidget.ts [hunk 1]
```diff
@@ -3,13 +3,20 @@
...
 
 export type DashboardWidget = {
-  title: string;
   type:
-    | 'line'
-    | 'scatter'
-    | 'bar'
-    | 'status-grid'
-    | 'status-timeline'
+    | 'line-chart'
```
Comment (boweihan) [inline apps/client/src/services/generated/models/DashboardWidget.ts:7]: question: should this type field come from the dashboard package?

**Review by boweihan — APPROVED** [review]

File: apps/client/src/services/generated/models/DashboardWidget.ts (hunk 1 above)
Comment (tracy-french) [reply apps/client/src/services/generated/models/DashboardWidget.ts:7]: If we have a good way to import the types into Core, yes. I don't think we should rely on two types.

File: package.json [hunk 2]
```diff
@@ -21,6 +21,8 @@
     "format:gen:types": "prettier --write \"${npm_package_config_genpath}/**/*.ts\"",
     "lint": "turbo run lint && tsc --noEmit && TIMING=1 eslint \"tests/**/*.ts\" --max-warnings 0",
     "lint:fix": "turbo run lint:fix && eslint \"tests/**/*.ts\" --fix",
+    "start:client": "cd apps/client && yarn dev && cd -",
+    "start:core": "cd apps/core && yarn dev && cd -",
```
Comment (chejimmy) [inline package.json:25]: Cleaner to call by workspaces, e.g. `"yarn workspace client dev"`

File: tests/dashboards/dashboard-management.spec.ts [hunk 3]
```diff
@@ -110,73 +79,3 @@ test('as a user, I can delete a dashboard', async ({ page }) => {
     'Successfully deleted dashboard "My Dashboard".',
   );
 });
-
-test('as a user, I can delete multiple dashboards', async ({ page }) => {
```
Comment (chejimmy) [inline tests/dashboards/dashboard-management.spec.ts:114]: don't need to cover this use case?

**Review by chejimmy — APPROVED** [review]
🚀🚀🚀 TY for completing the app

File: tests/dashboards/dashboard-management.spec.ts (hunk 3 above)
Comment (tracy-french) [reply tests/dashboards/dashboard-management.spec.ts:114]: Yes, but perhaps not as a UI test. I'm thinking going beyond the basic workflows with UI tests is not working out. I'll prioritize backfilling the test.

File: package.json (hunk 2 above)
Comment (tracy-french) [reply package.json:25]: Oh duh! I'll prioritize fixing this.

**Pull request merged**